### ISSUE TITLE
how do i install it is their a guide?

### ISSUE THREAD

**Issue by kylegotdrip12** [moom825/xeno-rat#88]

**Comment by NFS11** [comment]
can you be more clear with your question since I don't undestand

**Comment by kylegotdrip12** [comment]
how do i install xeno rat how do i make it work and send logs

**Comment by NFS11** [comment]
@kylegotdrip12  that was quick... anyway I suggest checking issue #15 and also build client set up a port and use that same port in listener. like I said, check issue 15

**Comment by kylegotdrip12** [comment]
thanks but what port ? i do not know how to do that

**Comment by NFS11** [comment]
@kylegotdrip12  it can  be any port just dont use 80 or 4444 because thats gonna trigger av detection and put you at risk of having the connection between you and the client intercepted which may lead to an ip leak so use other ports. make sure you have port forwarding setup. if you do not know how to do this, google "how to port forward on [insert your isp/router]". you should be good to go. let me know if you got any problems or questions.

**Comment by kylegotdrip12** [comment]
do you have discord by chance?

**Comment by NFS11** [comment]
@kylegotdrip12  yes but id rather not commnicate via discord so as to avoid a ban
